index.js

```diff
@@ -196,9 +257,15 @@ function getGermanCars(/* code here */) {
  *   return num * 2
  * }
 */
-const sum = null; // code here!
-const addFive = null; // code here!
-const argTimesTwo = null; // code here!
+const sum = ((a, b) => {
+  return a + b
+});
+const addFive = (num => {
+  return num +5
+});
+const argTimesTwo = (num => {
+  return num * 2
+});
 
```

Comment:
lines 260 to 268: When using arrow functions that are this simple you don't need to add the return. Arrow functions automatically add return for you. So you can remove the `return` word from each of these and also the curly braces. so line 260 should look like `const sum = ( a,b ) => a + b;`